regression/scripts/AVL/avl_files/aircraft.avl

```diff
@@ -1,167 +0,0 @@
-cruise
```

Comment:
Why is this AVL script deleted?

Reply:
Idk...why, but the regression/travis works.... @mclarke2 might be able to comment on the use of it.